test/suite/test_prefetch02.py

```diff
@@ -110,7 +110,7 @@ def test_prefetch_scenarios(self):
                         'key_format={},value_format={}'.format(self.key_format, self.value_format))
         c1 = s.open_cursor(self.uri)
         s.begin_transaction()
-        for i in range(1, self.nrows):
```

Comment:
Can we increase the cache size instead of change this?

Reply:
yep, that seems to help as well